src/Bin/OpenEnroth/CMakeLists.txt

```diff
@@ -14,7 +14,25 @@ if(OE_BUILD_PLATFORM STREQUAL "android")
     target_link_libraries(main PUBLIC application library_cli library_platform_main library_stack_trace Glob)
     target_link_options(main PRIVATE "-Wl,--version-script=${CMAKE_CURRENT_SOURCE_DIR}/libmain.map")
 else()
-    add_executable(OpenEnroth MACOSX_BUNDLE ${BIN_OPENENROTH_HEADERS} ${BIN_OPENENROTH_SOURCES})
+    if (WIN32)
+        add_executable(OpenEnroth MACOSX_BUNDLE ${BIN_OPENENROTH_HEADERS} ${BIN_OPENENROTH_SOURCES} ${CMAKE_CURRENT_SOURCE_DIR}/win_OpenEnroth.rc)
```

Comment:
No need for MACOSX_BUNDLE on windows?

Reply:
If I change it from OSX to WIN32 the retrace tests in release mode break

> [6/7] Completed 'OpenEnroth_TestData'
Could not set console input codepage to UTF-8
[6/7] C:\Windows\system32\cmd.exe /C "cd /D D:\a\OpenEnroth\OpenEnroth\build\test\Bin\GameTest && D:\a\OpenEnroth\OpenEnroth\build\src\Bin\OpenEnroth\OpenEnroth.exe retrace --headless --ch[Press any key to close this window]eck-canonical --glob D:/a/OpenEnroth/OpenEnroth/build/test/Bin/GameTest/test_data/data/*.json"
FAILED: test/Bin/GameTest/CMakeFiles/RetraceTest_Headless D:/a/OpenEnroth/OpenEnroth/build/test/Bin/GameTest/CMakeFiles/RetraceTest_Headless 
C:\Windows\system32\cmd.exe /C "cd /D D:\a\OpenEnroth\OpenEnroth\build\test\Bin\GameTest && D:\a\OpenEnroth\OpenEnroth\build\src\Bin\OpenEnroth\OpenEnroth.exe retrace --headless --check-canonical --glob D:/a/OpenEnroth/OpenEnroth/build/test/Bin/GameTest/test_data/data/*.json"